app/app.js

```diff
@@ -46,6 +47,7 @@ app.use((req, res, next) => {
       govukTemplate:
         '../../vendor/govuk_template_mustache_inheritance/views/layouts/govuk_template',
       googleTagManager: 'partials/google-tag-manager',
+      breadcrumbTemplate: 'partials/breadcrumb',
```

Comment:
This is technically a partial. Maybe just rename to breadcrumb to fit in with existing partial naming convention?